Alignment and Pretoucher issue on M1 mac

test have been ignored on M1 Mac, due to possible Alignment issue.

To access the M1 mac in the TC cluster - see https://docs.google.com/document/d/1-nsInZ9X4YDDrl_3te5RhQRgd-f6e68dVWC2h804M-s/edit

effecting the following tests : ( see https://teamcity.chronicle.software/buildConfiguration/OpenHFT_ChronicleQueue4_SnapshotMac/566375 ) 

```
StuckQueueTest.testnet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchnet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchLongBreaknet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchLeapnet.openhft.chronicle.queue.impl.single
```

for example:
```
StuckQueueTest.testnet.openhft.chronicle.queue.impl.single

Stacktrace
Copy to clipboard
java.lang.AssertionError: mis-aligned
  at net.openhft.chronicle.queue.impl.single.StuckQueueTest.test(StuckQueueTest.java:63)
------- Stdout: -------
--- !!meta-data #binary
header: !SCQStore {
wireType: !WireType BINARY_LIGHT,
writePosition: [
0,
0
],
roll: !SCQSRoll {
length: !int 60000,
format: yyyyMMdd-HHmm,
epoch: 0
},
indexing: !SCQSIndexing {
indexCount: !short 2048,
indexSpacing: 16,
index2Index: 442,
lastIndex: 0
```

also this test ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata  - see https://teamcity.chronicle.software/viewLog.html?buildId=566375&tab=buildResultsDiv&buildTypeId=OpenHFT_ChronicleQueue4_SnapshotMac

```
ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata
--
First failure: #4104  Changes (44) 20 Apr 21 12:09java.lang.AssertionError: mis-aligned 	at net.openhft.chronicle.queue.ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileInner(ChronicleQueueIndexTest.java:86) 	at net.openhft.chronicle.queue.ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata(Chro```

--- a/src/test/java/net/openhft/chronicle/queue/ChronicleQueueIndexTest.java
+++ b/src/test/java/net/openhft/chronicle/queue/ChronicleQueueIndexTest.java
@@ -39,9 +39,6 @@ public class ChronicleQueueIndexTest extends ChronicleQueueTestBase {
     @Test
     public void checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata() {
 
-        // todo remove see https://github.com/OpenHFT/Chronicle-Queue/issues/837
-        Assume.assumeTrue(!Jvm.isMacArm());
-
         final Consumer<InternalAppender> writer = appender -> {
             try (DocumentContext wd = appender.writingDocument(true)) {
                 wd.wire().write("key").writeDouble(1);

--- a/src/test/java/net/openhft/chronicle/queue/impl/single/StuckQueueTest.java
+++ b/src/test/java/net/openhft/chronicle/queue/impl/single/StuckQueueTest.java
@@ -24,8 +24,7 @@ public class StuckQueueTest extends ChronicleQueueTestBase {
     @Test
     public void test() throws IOException {
 
-        // todo remove see https://github.com/OpenHFT/Chronicle-Queue/issues/837
-        assumeFalse(Jvm.isMacArm());
+
         // java.nio.file.InvalidPathException: Illegal char <:> at index 2: /D:/BuildAgent/work/1e5875c1db7235db/target/test-classes/stuck.queue.test/20180508-1249.cq4
         assumeFalse(OS.isWindows());
 

